packages/database/seeders/20221104020243-default-course-schedules.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.bulkInsert(
+      "ScheduleWeekDays",
+      [
+        {
+          monday: true,
+          tuesday: false,
+          wednesday: true,
+          thursday: false,
+          friday: true,
+          saturday: false,
+          sunday: false,
+        },
+      ],
+      {}
+    );
+    await queryInterface.bulkInsert("Schedules", [
+      {
+        startDate: "2022-11-08",
+        endDate: "2023-02-08",
+        startTime: "20:00:00",
+        endTime: "23:00:00",
+        /*Consultar como se combina el S.W.D con Schedule*/
+        scheduleWeekDaysId: null,
```

Comment:
You will just use the ScheduleWeekDaysId here. It should be 1 since you are introducing the first in top of this document
```suggestion
        scheduleWeekDaysId: 1,
```

Reply:
you're right!!